diff --git a/dist/mdx-plugins/remark-replace-imports.mjs b/dist/mdx-plugins/remark-replace-imports.mjs
index 2d1bd6cf632116dd71c2f38d627b5d5248e3f4be..2a41d7fadf6e92b7a9e966575434e0c6cfbda8c7 100644
--- a/dist/mdx-plugins/remark-replace-imports.mjs
+++ b/dist/mdx-plugins/remark-replace-imports.mjs
@@ -1,18 +1,18 @@
 // src/mdx-plugins/remark-replace-imports.ts
-import { createRequire } from "module";
-import { visit } from "unist-util-visit";
-var require2 = createRequire(import.meta.url);
+import { createRequire } from 'module'
+import { visit } from 'unist-util-visit'
+var require2 = createRequire(import.meta.url)
 var remarkReplaceImports = () => {
   return (tree, _file, done) => {
-    visit(tree, "mdxjsEsm", (node) => {
-      const { source } = node.data.estree.body[0];
-      const absolutePath = require2.resolve(source.value);
-      source.value = absolutePath;
-      source.raw = `"${absolutePath}"`;
-    });
-    done();
-  };
-};
-export {
-  remarkReplaceImports
-};
+    visit(tree, 'mdxjsEsm', (node) => {
+      const { source } = node.data.estree.body[0]
+      if (source) {
+        const absolutePath = require2.resolve(source.value)
+        source.value = absolutePath
+        source.raw = `"${absolutePath}"`
+      }
+    })
+    done()
+  }
+}
+export { remarkReplaceImports }
